src/pdb.rs

```diff
@@ -512,13 +512,11 @@ impl DeviceSQLString {
         let (input, length) = nom::number::complete::le_u16(input)?;
         let (input, _) = nom::bytes::complete::tag(b"\x00")(input)?;
 
-        // TODO: Ensure that length is always even
-        //assert_eq!(length % 2, 0);
-        // Don't include the trailing two nullbytes in the string.
-        let str_length = usize::from(length) / 2 - 1;
+        // sanity check
```

Comment:
I think the previous comment was more descriptive tbh.

```suggestion
        //  Ensure that length is always even.
```

Reply:
IMO it doesnt add much value either. `length % 2` is self descriptive. A better comment would be to explain why we actually ensure this.
```suggestion
        // The length is in bytes, UTF-16 code points are always 2 bytes wide, so a valid length must be even. 
```